install.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
speaking of bash, this needs to be changed to `#!/usr/bin/env bash`, or else it won't work at all on some distros (like nix, which is supported)